README.md

```diff
@@ -6,18 +6,22 @@ Apiary provides modules which can be combined to create a federated cloud data l
 * Read/write [Hive](https://hive.apache.org) metastore service
 * Read only [Hive](https://hive.apache.org) metastore service
 * [Waggle Dance](https://github.com/HotelsDotCom/waggle-dance) federated Hive metastore service
+* [Shunting Yard](https://github.com/ExpediaGroup/shunting-yard) event-based data replication service
 * Related infrastructure including load balancers
 * Various extensions and plugins for adding additional functionality to the Hive metastore
 
 ## Components
 Apiary consists of the following components which are managed in separate git repositories:
 * [Apiary Authorization](https://github.com/ExpediaGroup/apiary-authorization)
 * [Apiary Data Lake](https://github.com/ExpediaGroup/apiary-data-lake)
-* [Apiary Extensions](https://github.com/ExpediaGroup/apiary-extensions)
+* [Apiary Replication](https://github.com/ExpediaGroup/apiary-replication)
 * [Apiary Federation](https://github.com/ExpediaGroup/apiary-federation)
 * [Apiary Metastore Docker](https://github.com/ExpediaGroup/apiary-metastore-docker)
 * [Apiary Ranger Docker](https://github.com/ExpediaGroup/apiary-ranger-docker)
 * [Apiary Waggle Dance Docker](https://github.com/ExpediaGroup/apiary-waggledance-docker)
+* [Apiary Shunting Yard Docker](https://github.com/ExpediaGroup/apiary-shuntingyard-docker)
```

Comment:
Looks as though these were in alphabetical order, perhaps we should keep them that way for readability?

Reply:
They do ! But I feel that might be a coincidence. I felt the intention was to arrange it as follows:

main components
docker images of those components.

I just combined the extensions and its terraform at the end together. Happy to make it alphabetical if thats how we want to keep.

Reply:
yes they are in alphabetical order.

Reply:
Alright. done.